tests/test_hooks.sh

```
@@ -243,83 +256,85 @@ echo "=== review-to-rules-gate.sh ==="
 write_vibecorp_yml
 
 # 1. スタンプなしで gh pr merge → deny
-rm -f ${TMPDIR_ROOT}/.claude/state/review-to-rules-ok
+rm -f ${STAMP_REVIEW_TO_RULES}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)